config/index.js

```diff
@@ -17,6 +17,11 @@ var conf = convict({
     format: [ 'dev', 'test', 'stage', 'prod' ],
     env: 'NODE_ENV'
   },
+  geodb: {
+    doc: 'Config for geodb location service',
+    dbPath: path.resolve(__dirname, '../node_modules/fxa-geodb/db/cities-db.mmdb'),
+    shouldReturnGeodata: true
+  },
```

Comment:
@TDA This is not correct. 

See example: https://github.com/mozilla/fxa-auth-server/blob/master/config/index.js#L31-L49